src/test/java/teammates/test/AbstractBackDoor.java

```diff
@@ -869,6 +869,13 @@ public DeadlineExtensionAttributes getDeadlineExtension(
                 .build();
     }
 
+    /**
+     * Sync feedback session logs from GCloud Logging Service.
```

Comment:
"GCloud Logging Service" is an implementation detail that does not matter for BackDoor. It should not be mentioned here.